com/zoolatech/java/solomiia_tymoshchuk/src/lecture1/tasks/_8/Task8.java

```diff
@@ -0,0 +1,60 @@
+package com.zoolatech.java.solomiia_tymoshchuk.src.lecture1.tasks._8;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Write a program that prints all numbers from 1 to N, that are divisible by 2, 3 or by both.
+ * If a number is divisible by only 2 - print it and add in parentheses “(by 2)”,
+ * if only by 3 - “(by 3)”, if by both numbers - “(by 2 and 3)”.
+ * Every number should be printed on the new line.
+ * Create two versions of the program: one should use the for loop, second - the while loop.
+ **/
+public class Task8 {
+    int number;
+    public Task8(int number){
+        this.number = number;
+
+    }
+    List<Integer> newarr = new ArrayList<>();
+
+    //for loop version of program
+    void checkDivisionForLoop() {
+        for (int i = 1; i <= number; i++) {
```

Comment:
```
        for (int i = 1; i <= number; i++) {
            newarr.add(i);
        }
```

can be removed - you're adding values to the list, but not read them

same for
```
        int i = 1;
        while (i <= number) {
            newarr.add(i);
            i++;
        }
```
in checkDivisionWhileLoop